index.js

```diff
@@ -120,3 +146,10 @@ console.log('hello world')
  */
 
 // TODO: ADD CODE HERE
+multiplyByTwo = (number) => number * 2;
```

Comment:
You should use a variable keyword when declaring a new function (i.e. `const multiplyByTwo = (number) => {...}`)